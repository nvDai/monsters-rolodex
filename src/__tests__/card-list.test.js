import React from 'react'

import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import CardList from '../components/card-list/card-list.component'

describe('CardList', () => {
  it('should render without crashing', () => {
    const mockProps = {
      monsters: [
        {
          id: '1',
          name: 'Michael',
          email: 'michael@gmail.com'
        },
        {
          id: '2',
          name: 'Michael 2',
          email: 'michael2@gmail.com'
        }
      ]
    }
    const wrapper = shallow(<CardList {...mockProps} />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})