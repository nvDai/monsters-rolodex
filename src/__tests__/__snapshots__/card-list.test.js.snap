// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList should render without crashing 1`] = `
<div
  className="card-list"
>
  <Card
    key="1"
    monster={
      Object {
        "email": "michael@gmail.com",
        "id": "1",
        "name": "Michael",
      }
    }
  />
  <Card
    key="2"
    monster={
      Object {
        "email": "michael2@gmail.com",
        "id": "2",
        "name": "Michael 2",
      }
    }
  />
</div>
`;
